### PR TITLE
finish task 6 and tests

### PR DESCRIPTION
**Pull request by karlrivkpmg**

### CODE REVIEW

File: server/__tests__/app.test.js [hunk 1]
```diff
@@ -114,3 +113,58 @@ afterAll(() => {
...
+
+  describe('5. GET /api/reviews/:review_id/comments', () =>{
+
+    test("status:200, returns an array of comments matching the parametric review_id", ()=>{
+        return request(app)
+        .get('/api/reviews/2/comments')
+        .expect(200)
+        .then((response)=>{
+            const comments = response.body.comments;
+            expect(true).toBe(Array.isArray(comments));
```
Comment (RichardNeat) [inline server/__tests__/app.test.js]: we don't need to explicitly check for an array here, as we are using forEach later it is implied that it must be an array

File: server/__tests__/app.test.js [hunk 2]
```diff
@@ -114,3 +113,58 @@ afterAll(() => {
...
+                        created_at: expect.any(String),
+                        votes: expect.any(Number)
+                    })
+                )
+            }) 
+            expect(comments).toBeSortedBy('created_at', {descending: true});
+        })
+    })
+
+    test("status:200, returns an empty array matching the parametric review_id", ()=>{
```
Comment (RichardNeat) [inline server/__tests__/app.test.js]: I would add to the description that this is a case where the id is valid and exists but there happens to be no comments for it

**Review by RichardNeat — COMMENTED** [review]
Nice one Karl, just a couple of minor changes before merging please, then go for it :)

File: server/__tests__/app.test.js (hunk 2 above)
Comment (karlrivkpmg) [reply server/__tests__/app.test.js]: great rich cheers

**Pull request merged**